src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -299,7 +291,6 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'multiple' => false,
             'expanded' => false,
-            'choice_list' => null, // deprecated
             'choices' => array(),
             'choices_as_values' => false,
```

Comment:
this option should be deprecated saying it has no effect anymore (and the value should be ``true`` to reflect the behavior of the code).
The option should be kept though, to allow writing code compatible with 2.8 and 3.0 (passing the value explicitly)